tests/components/bang_olufsen/test_media_player.py

```diff
@@ -174,13 +194,13 @@ async def test_async_update_playback_error(
     mock_config_entry.add_to_hass(hass)
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
 
+    playback_error_callback = (
+        mock_mozart_client.get_playback_error_notifications.call_args[0][0]
+    )
+
     # The async_dispatcher_send function seems to swallow exceptions, making pytest.raises unusable
     with patch("homeassistant.helpers.dispatcher._LOGGER.error") as mock_logger:
```

Comment:
In the future I think we should refactor this to use a `caplog` and just check if what we expect is put in the log is put there

Reply:
We removed these right